### PR TITLE
hsw-cli: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/hsw-cli.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`hsw-cli balance`
+
+- View the current wallets transaction history:
+
+`hsw-cli history`
+
+- Send a transaction to the specified address:
+
+`hsw-cli send {{address}} {{value}}`
```
Comment (sbrl) [inline pages/common/hsw-cli.md]: What kind of value is this? Perhaps a concrete example might be better here.

Comment (pxgamer) [reply pages/common/hsw-cli.md]: It's the amount to send. So, `1.03` for example.

I guess maybe `coin_amount` may be more fitting?

File: pages/common/hsw-cli.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# hsw-cli
+
+> The command line REST tool for the Handshake wallet.
+> More information: <https://npmjs.com/package/hs-client>.
+
+- Unlock the current wallet:
+
+`hsw-cli unlock {{passphrase}} {{timeout}}`
```
Comment (agnivade) [inline pages/common/hsw-cli.md]: What is the unit of timeout ? Maybe good to mention that in the description.

File: pages/common/hsw-cli.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Lock the current wallet:
+
+`hsw-cli lock`
+
+- View the current wallets details:
```
Comment (agnivade) [inline pages/common/hsw-cli.md]: wallet's ? And same for the rest.

File: pages/common/hsw-cli.md (hunk 1 above)
Comment (agnivade) [reply pages/common/hsw-cli.md]: I think directly mentioning 1.03 might be simpler. And mention "coin_amount" in the description.

File: pages/common/hsw-cli.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`hsw-cli send {{address}} {{value}}`
+
+- View the current wallets pending transactions:
+
+`hsw-cli pending`
+
+- View details about a transaction:
+
+`hsw-cli tx {{transaction_hash}}`
```
Comment (agnivade) [inline pages/common/hsw-cli.md]: How do I get a list of transactions ? Using `hsw-cli tx` ? We should also mention that.

File: pages/common/hsw-cli.md (hunk 2 above)
Comment (pxgamer) [reply pages/common/hsw-cli.md]: It's in seconds, I'll add that to the description.

File: pages/common/hsw-cli.md (hunk 4 above)
Comment (pxgamer) [reply pages/common/hsw-cli.md]: `hsw-cli history`, it's one of the other examples. :+1:

**Pull request merged**